sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/parallel.py

```diff
@@ -266,9 +266,9 @@ def _attr_type_map(cls) -> dict:
             "resources": (dict, JobResourceConfiguration),
             "task": (dict, ParallelTask),
             "logging_level": str,
-            "max_concurrency_per_instance": int,
-            "error_threshold": int,
-            "mini_batch_error_threshold": int,
+            "max_concurrency_per_instance": (str, int),
```

Comment:
this should be the only necessary change. Please add test and check if you can revert changes other than attr_map

Reply:
reverted the schema change